conf/application.conf

```diff
@@ -54,6 +54,10 @@ tracking-consent-frontend {
   gtm.container = "a"
 }
 
+mongodb {
+    uri = "mongodb://localhost:27017/gform"
```

Comment:
Shouldn't we call this database `gform-frontend` to match service name and prevent potential future confusion?